addons/mail/tests/test_mail_render.py

```diff
@@ -255,20 +263,20 @@ def cust_function():
         cust_function.call = False
 
         src = """<h1>This is a test</h1>
-<p>${cust_function()}</p>"""
+<p>{{ cust_function() }}</p>"""
         expected = """<h1>This is a test</h1>
 <p>return value</p>"""
         context = {'cust_function': cust_function}
 
-        result = self.env['mail.render.mixin'].with_user(self.user_admin)._render_template_jinja(
+        result = self.env['mail.render.mixin'].with_user(self.user_admin)._render_template_inline_template(
             src, partner._name, partner.ids,
             add_context=context
         )[partner.id]
         self.assertEqual(expected, result)
         self.assertTrue(cust_function.call)
 
-        with self.assertRaises(AccessError, msg='Simple user should not be able to render Jinja code'):
-            MailRenderMixin._render_template_jinja(src, model, res_ids, add_context=context)
+        with self.assertRaises(AccessError, msg='Simple user should not be able to render inline_template code'):
```

Comment:
"... to render dynamic code" ? or "template-based code" ? inline_template is not a work you want to show to standard people.